### DISCUSSION TITLE
Can you fix bug for me?

### DISCUSSION THREAD

**Discussion by duykhanh2612** [duykhanh2612/Portfolio_Projects#2] [Q&A] [answered]
Do you think I need to split or merge distinct projects each other.
And can you look through my project and give me some advice from that? 
![image](https://github.com/user-attachments/assets/7f54f00e-a1f3-432c-8c85-beafabea38e6)
Can you explain this chart for me? Submit the question below

**Comment by vthuhien** [comment, marked as answer]
Hi, 
I think you need to split clearly the code and I will modify, add and commit change for your project after
Now, I will define the chart for you. The chart indicates that we have 19 columns with missing values. It seems to be empty values ​rather than lost. The code for this.
total = df.isnull().sum().sort_values(ascending = False)
percent = (total/df.isnull().count().sort_values(ascending = False))
board = pd.concat([total, percent], axis = 1, keys = ['total', 'percent'])
board = board[(board['total']>0) & (board['percent']>=0)]
board
![image](https://github.com/user-attachments/assets/b9b98cd8-c870-4226-9232-44d2d2875782)
